src/pages/MoviePage.js

```diff
@@ -1,10 +1,130 @@
 import React from "react";
-
```

Comment:
I'll keep this Pull Request open to get back to it later, to solve the sizing issues that I've sent to you on Telegram